packages/xrpl/test/models/CredentialCreate.test.ts

```
@@ -0,0 +1,230 @@
+      Sequence: 1337,
+      Flags: 0,
+    } as any
+  })
+
+  it(`verifies valid credentialCreate`, function () {
+    assert.doesNotThrow(() => validateCredentialCreate(credentialCreate))
+    assert.doesNotThrow(() => validate(credentialCreate))
+  })
+
+  it(`throws w/ missing field Account`, function () {
+    delete credentialCreate.Account
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)